AirCasting/SessionViews/ShareSession/CSVFileGenerator.swift

```diff
@@ -0,0 +1,31 @@
+// Created by Lunar on 12/01/2022.
+//
+
+import Foundation
+
+protocol CSVFileGenerator {
+    func generateFile(content: String, fileName: String) -> Result<URL, Error>
+}
+
+struct DefaultCSVFileGenerator: CSVFileGenerator {
+    func generateFile(content: String, fileName: String) -> Result<URL, Error> {
+        let fileManager = FileManager.default
+        do {
+            let path = try fileManager.url(for: .documentDirectory, in: .allDomainsMask, appropriateFor: nil, create: false)
```

Comment:
hmm, first time I see this approach, usually to get documents dir we do:
`FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)[0]`